src/include/pbs_internal.h

```diff
@@ -391,11 +391,12 @@ enum accrue_types {
 #endif
 #define LOCALHOST_SHORTNAME "localhost"
 #ifdef WIN32
-#define CLOSESOCKET(X) (void)closesocket(X)
 #define ERRORNO        WSAGetLastError()
+#define initsocketlib() winsock_init()
```

Comment:
How about moving this to pbs_config.h ?

Reply:
I agree this is not quite a right place. but since pbs_config.h is dynamically generated in some platforms it will not be possible to move it there.  May be we can find a clean place in next PRs, for now we can keep it here. Is it agreeable?

Reply:
I think I found a better place and moved it there, see https://github.com/openpbs/openpbs/pull/1917/commits/7efec9bf92b4900035dd88946aecda941176c6be